### ISSUE TITLE
View list of discussions

### ISSUE THREAD

**Issue by nueliza** [nueliza/ENPM613#38]
As a student or tutor, I should be able to see the list of discussions in the system

**Comment by nueliza** [comment]
Tutor -> mod_id is asked for getting discussion List.
Student -> completed

**Comment by aarohimehta** [comment]
<img width="1138" alt="Screen Shot 2019-11-23 at 4 06 35 PM" src="https://user-images.githubusercontent.com/54917968/69485498-6da3be00-0e0e-11ea-881b-e4e6eabb0822.png">
As a tutor get discussion without mod_id

**Comment by nueliza** [comment]
Thanks @aarohimehta !

**Comment by nueliza** [comment]
@aarohimehta , Though the GET request shouldn't take anything in the body, for this particular request, when the body is empty, it throws a 400/ 500

![image](https://user-images.githubusercontent.com/17264279/69511405-f35a6300-0f0d-11ea-9318-57a5157b52a1.png)

**Comment by aarohimehta** [comment]
Yeah, you are correct. Getting the same error too. Wiill raise issue in Ujjwal's Repo
